.github/workflows/preview_on_pr_update.yml

```diff
@@ -208,15 +208,15 @@ jobs:
     runs-on: macos-14 
     timeout-minutes: 30
     env:
+      ENV: preview
       NO_FLIPPER: 1 # When set to 1, skips installing all Flipper-related dependencies so they’re removed from the build
-      IOS_MATCH_PASSWORD: ${{ secrets.IOS_MATCH_PASSWORD }}
-      IOS_MATCH_REPOSITORY_URL: ${{ secrets.IOS_MATCH_REPOSITORY_URL }}
-      IOS_APPLE_ID: ${{ secrets.IOS_APPLE_ID }}
+      IOS_MATCH_PASSWORD: ${{ secrets.IOS_MATCH_PASSWORD_PREVIEW }}
```

Comment:
We want to keep same variable name `IOS_MATCH_PASSWORD_PREVIEW` should be `IOS_MATCH_PASSWORD` but the value should change for the config.

Same for other variables.